src/compiler/utils/exec_sync_pretty.ts

```diff
@@ -0,0 +1,9 @@
+import { execSync, IOType } from 'child_process';
+
+export function execSyncPretty(command: string, stdio?: IOType): Buffer {
```

Comment:
Could we update the type of stdio to be IOType | IOType[]?

Reply:
Use case would be in dfx/index.ts

Reply:
Done